src/main/java/org/jabref/logic/importer/fetcher/CompositeSearchBasedFetcher.java

```diff
@@ -60,7 +60,6 @@ public String getName() {
                            }
                        })
                        .limit(maximumNumberOfReturnedResults)
-                       .map(cleanup::doPostCleanup)
```

Comment:
the `cleanup` variable is still declared above but now unused if I'm not mistaken